app/routes/index.js

```diff
@@ -103,20 +120,42 @@ const drawSectionGetRequests = (section) => {
   })
 }
 
+const getGrantValues = (projectCost) => {
+  const calculatedGrant = Number(GRANT_PERCENTAGE * projectCost / 100).toFixed(2)
+  const remainingCost = Number(projectCost - calculatedGrant).toFixed(2)
+  return { calculatedGrant, remainingCost }
+}
+
 const getPostHandler = (currentQuestion) => {
   const { yarKey, answers, url, ineligibleContent, nextUrl, maybeEligibleContent } = currentQuestion
   const MAYBE_ELIGIBLE = { url, nextUrl, maybeEligibleContent }
   const NOT_ELIGIBLE = { url, ineligibleContent }
-
   return (request, h) => {
     const value = request.payload[Object.keys(request.payload)[0]]
+    const { calculatedGrant, remainingCost } = getGrantValues(value)
+    const checkNotEligible = (answer) => {
+      if (answer.value === value && !answer.isEligible) {
+        return true
+      } else if ((calculatedGrant < MIN_GRANT) || (calculatedGrant > MAX_GRANT)) {
+        return true
+      }
+    }
+    const checkMaybeEligible = (answer) => {
```

Comment:
if this function returns a boolean, there must be a case when it returns ``` false```
What happens if neither condition A (line 144) nor B (line 146) is met?